sktime/classification/deep_learning/rnn.py

```diff
@@ -66,7 +67,16 @@ def __init__(
     ):
         _check_dl_dependencies(severity="error")
         super().__init__()
-        self.num_epochs = num_epochs
+        # Deprecated Parameter Handling
+        if num_epochs is not None:
+            warnings.warn(
+                "The parameter 'num_epochs' is deprecated and will be removed in version 0.26.0. "
```

Comment:
Very good! Please mention the name of the estimator too, i.e., "in estimatorname, ..."